### DISCUSSION TITLE
prime react bread crumb separator icon are not appearing

### DISCUSSION THREAD

**Discussion by shivashankara** [primefaces/community#1389] [PrimeReact] [answered]
prime react bread crumb separator icon are not appearing please help me to fins the root cause and fix the issue

**Comment by Latonya86Dodson** [comment]
Hello,

One option is to use the separator prop of the BreadCrumb component to customize the separator icon. You can pass a string, a React element, or a function that returns a React element as the separator prop. For example, you can use the following code to use a right arrow icon as the separator:

import { BreadCrumb } from 'primereact/breadcrumb';
import { FaArrowRight } from 'react-icons/fa';

const items = [
  { label: 'Categories' },
  { label: 'Sports' },
  { label: 'Football' },
  { label: 'Countries' },
  { label: 'Spain' },
  { label: 'F.C. Barcelona' },
  { label: 'Squad' },
  { label: 'Lionel Messi', url: 'https://en.wikipedia.org/wiki/Lionel_Messi[BeBallPlayers](https://www.beballplayers.com/)' }
];

const home = { icon: 'pi pi-home', url: 'https://www.primefaces.org/primereact' }

const MyBreadCrumb = () => {
  return (
    <BreadCrumb model={items} home={home} separator={<FaArrowRight />} />
  );
};

I hope this helps you.

**Reply by melloware** [reply, marked as answer]
It works fine see this StackBlitz: https://stackblitz.com/edit/8tzga6?file=src%2FApp.jsx

Compare with what you are doing...
